.github/workflows/test.yml

```diff
@@ -6,26 +6,35 @@ on:
     inputs:
       eve_image:
         type: string
-      runner:
-        type: string
-        default: buildjet-4vcpu-ubuntu-2204
   workflow_call:
     inputs:
       eve_image:
         type: string
-      runner:
-        type: string
-        default: buildjet-4vcpu-ubuntu-2204
 
 jobs:
+  determine-runner:
+    name: Determine best available runner
+    runs-on: ubuntu-latest
+    outputs:
+      runner: ${{ steps.fork-check.outputs.runner }}
+    steps:
+      - id: fork-check
+        run: |
+          if ["${{ github.event.repository.full_name}}" == "lf-edge/eve" || "${{ github.event.repository.full_name}}" == "lf-edge/eden"]; then
+            echo "runner=buildjet-4vcpu-ubuntu-2204" >> "$GITHUB_OUTPUT"
+          else
+            echo "runner=ubuntu-2204" >> "$GITHUB_OUTPUT"
```

Comment:
It was `ubuntu-22.04` before. Also please solve https://github.com/lf-edge/eden/pull/909#discussion_r1366923094